### DISCUSSION TITLE
NextDNS DNS-over-TLS/QUIC forwarder doesn't work

### DISCUSSION THREAD

**Discussion by c-michail** [TechnitiumSoftware/DnsServer#961] [General]
Hello,
  If i set NextDNS DNS-over-TLS/QUIC it doesn't work (windows 11) but the DNS-over-HTTPS forwarder works.
  NextDNS DNS-over-TLS/QUIC: xxxxx.dns.nextdns.io (where xxxxx is your profile number)

Any ideas what might be wrong ?

**Comment by ShreyasZare** [comment]
Thanks for the post. Please check the DNS logs to find any errors related to it. The error log will explain what went wrong. You can post the error log here if you need help understanding that. If you do not wish to post it here then send it to support@technitium.com.

**Comment by c-michail** [comment]
Hello,
  I sent the logs to support@technitium.com

Thank you

**Reply by ShreyasZare** [reply]
Thanks for the post. I have not received the email yet. Please check if there was any typo in the email id while sending.

**Reply by ShreyasZare** [reply]
Got the email. Thanks for the logs. Most error logs indicate some kind of network issue causing the request to timeout when using QUIC. Not sure what could be the cause. It could also be a temporary issue which may go away after a while.

NextDNS does work with Technitium and you can test it from [dnsclient.net](https://dnsclient.net/#xxxxx.dns.nextdns.io/google.com/A/QUIC/false/) which uses the same implementation.

**Reply by c-michail** [reply]
Weird, i switch back to nextdns quic and now it's working.

Thank you

**Reply by c-michail** [reply]
I spoke too soon Server Failures with Quic went to the roof! Switched back to DoH

Log full of failures: [2024-07-27 18:56:32 UTC] DNS Server failed to resolve the request 'www.hero-wars.com. A IN' using forwarders: xxxxxx.dns.nextdns.io:853

![εικόνα](https://github.com/user-attachments/assets/28367e5a-fc03-48df-9b61-87633d1072c7)

**Reply by ShreyasZare** [reply]
It seems some kind of network issue at your end. It could also be some old middlebox like a router which has issue with QUIC protocol that comes up after a while of using it. In such case, using DoH or DoT would be better. DoT may work better in some cases than DoH.

**Reply by c-michail** [reply]
or NextDNS is limiting their queries requests. I'll try their business plan to see if will happen again.
